specification/elasticsan/resource-manager/Microsoft.ElasticSan/preview/2024-07-01-preview/elasticsan.json

```diff
@@ -654,6 +669,21 @@
           },
           {
             "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "purge",
+            "description": "Optional, used to purge soft deleted volume groups if set to true.",
```

Comment:
Does this mean it will be permanent deleted?
What's the default value? 
Please update the description to include these info and make the meaning of the parameter more clear to customer.

Besides that, if this is for permanent delete:
1. Permanent deleted blob parameter is hidden in storage SDK/PSH/CLI. Should we also hide this parameter in PSH/CLI/SDK?
2. Why not use same parameter name as blob permanent delete?

Reply:
What is the blob permanent delete parameter name?
For blob, is it just allowed through portal then?